npm/test/basics.js

```diff
@@ -337,11 +335,64 @@ test("cancel worker", () => {
   });
 });
 
+test("language service diagnostics", async () => {
+  const languageService = getLanguageService();
+  languageService.addEventListener("diagnostics", (event) => {
+    assert.equal(event.type, "diagnostics");
```

Comment:
How do you know this event listener even got called in this test?

Reply:
Thanks, that's an oversight. I'll add a check to make sure the callback got called.